lms/djangoapps/teams/tests/test_views.py

```diff
@@ -129,7 +129,7 @@ def test_query_counts(self):
         team.add_user(self.user)
 
         # Check the query count on the dashboard again
-        with self.assertNumQueries(19):
+        with self.assertNumQueries(20):
```

Comment:
@wajeeha-khalid Since we're focusing on the user_api endpoint right now, would you mind adding a `self.assertNumQueries` check in a success test method for GET AccountView?  I'd like to make sure the number of SQL queries triggered by the mobile app's API call is reasonable.